src/Reflection/ReflectionClass.php

```diff
@@ -501,9 +501,9 @@ public function hasMethod(string $methodName): bool
      *
      * @return array<string, scalar|array<scalar>|null>
      */
-    public function getImmediateConstants(): array
+    public function getImmediateConstantsValues(): array
```

Comment:
IMO, let's drop this completely - let the userland consumer do the `array_map()` operations